pages/community/index.js

```diff
@@ -0,0 +1,340 @@
+/* eslint-disable react/no-unescaped-entities */
+import React from 'react';
+import Heading from '../../components/typography/Heading';
+import Button from '../../components/buttons/Button';
+import IconRocket from '../../components/icons/Rocket';
+import NewsletterSubscribe from '../../components/NewsletterSubscribe';
+import GenericLayout from '../../components/layout/GenericLayout';
+import orbitData from '../../config/orbitData.json';
+import { getEvents } from '../../lib/staticHelpers';
+import eventsData from '../../config/meetings.json';
+
+function CommunityIndexPage() {
+  const image = '/img/social/website-card.png';
+  const heading = (
+    <div className="text-center flex justify-center flex-col items-center mt-10 md:mt-0">
+      <Heading
+        className="countdown-text-gradient font-bold"
+        level="h6"
+        typeStyle="heading-xs"
+      >
+        AsyncAPI Community
+      </Heading>
+      <div className="mt-10">
+        <Heading level="h1" typeStyle="heading-xl" className="">
+          <span className="title block md:-mt-1 leading-[3rem]">
+            Welcome to the<br /> AsyncAPI Community
+          </span>
+        </Heading>
+      </div>
+      <div className="mt-5 w-5/6">
+        <Heading
+          level="h2"
+          typeStyle="body-md"
+          textColor="text-gray-700"
+          className="text-slate-500 text-sm"
+        >
+          We're an OSS community that's passionate about AsyncAPI.
+          Join us in building the future of Event Driven APIs by asking
+          questions, sharing ideas, and building connections.
+        </Heading>
+      </div>
+      <div className="mt-10">
+        <Button
+          className="block md:inline-block focus:outline-none"
+          text="AsyncAPI Discussions"
+          href="https://github.com/orgs/asyncapi/discussions"
+          target="_blank"
+          icon={<IconRocket className="w-5 h-5 -mb-1 ml-1" />}
+        />
+      </div>
+    </div>
+  );
```

Comment:
I see that you are making a separate variable to render this block. Will you think of making this as a separate issue?

Also, I see that most of the files inside `pages/communtiy` has huge lines of code. Can we think of making separate components for them as we are also looking for making UI tests on website?